package.json

```diff
@@ -5,8 +5,14 @@
   "main": "index.js",
   "scripts": {
     "start": "serve .",
```

Comment:
The `serve` package is not on your devDeps.

Reply:
Oops!